### PR TITLE
[Tooltip] Remove content max-height

### LINKED ISSUE #2869: Tooltip with long text seems t have no padding-bottom
## Issue summary

[Tooltip](https://polaris.shopify.com/components/overlays/tooltip) with long text seems t have no padding-bottom.

Screen Shot 2020-03-23 at 3 19 45 PM

You can open this [codesandbox](https://codesandbox.io/s/goofy-cdn-w2fbq) on Chrome or Safari to reproduce the bug.

### PR DESCRIPTION
**Pull request by thayannevls**
### WHY are these changes introduced?

Fixes #2869 

### WHAT is this pull request doing?
Increasing content and wrapper max-height to `500px` which is a great size for the tooltip in my opinion, If you guys don't consider this a good solution I'm open to change it.

**Before**
Screenshot from 2020-04-08 21-04-08

**After**
Screenshot from 2020-04-08 21-04-45

## 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris-react/blob/master/README.md#development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris-react/blob/master/documentation/Versioning%20and%20changelog.md)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';

import {Page} from '../src';
import {Tooltip, Link} from '../src'

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
      <Tooltip
        content="this is a loooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong text"
      >
        <Link>Order #1001</Link>
      </Tooltip>
    </Page>
  );
}
```
</details>

### CODE REVIEW

File: src/components/Tooltip/Tooltip.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import '../../styles/common';
 
 $offset-from-activator: rem(4px);
-$content-max-height: rem(100px);
+$content-max-height: rem(500px);
```
Comment (sylvhama) [inline src/components/Tooltip/Tooltip.scss:4]: I would even remove all max-height but maybe someone from Polaris should confirm.

Comment (kyledurand) [reply src/components/Tooltip/Tooltip.scss:4]: I agree to that sentiment. I would be sure that removing `max-height` looks as it should in all supported browsers first. Add me as a reviewer and I can help 🎩 on Monday if you need it

Comment (carolopolo) [reply src/components/Tooltip/Tooltip.scss:4]: +1 for that sentiment!

Comment (thayannevls) [reply src/components/Tooltip/Tooltip.scss:4]: I agree! Can you guys help me get the confirmation to do this? I'm not sure who is part of the Polaris core team

Comment (sylvhama) [reply src/components/Tooltip/Tooltip.scss:4]: @kyledurand is so you're good to go ^^

Comment (kyledurand) [reply src/components/Tooltip/Tooltip.scss:4]: I've just tophatted the removal of max-height from the container and content classes in Firefox, Chrome, Safari, mobile Safari, and chromium edge. All look good to me. If you could update this PR I'll approve 👍

Comment (thayannevls) [reply src/components/Tooltip/Tooltip.scss]: Awesome! Just updated, thanks.

File: UNRELEASED.md [hunk 2]
```diff
@@ -10,6 +10,7 @@
 ### Enhancements
 
 - Added utilities for parsing video duration (https://polaris.shopify.com/components/images-and-icons/video-thumbnail) ([#2725](https://github.com/Shopify/polaris-react/pull/2725))
+- Removed max-height property from `Tooltip`.
```
Comment (kyledurand) [inline UNRELEASED.md:13]:
```suggestion
- Removed max-height property from `Tooltip` [#2908](https://github.com/Shopify/polaris-react/pull/2908).
```

Don't forget to link to this PR in here as well

Comment (thayannevls) [reply UNRELEASED.md]: Done!

**Comment by thayannevls** [comment]
The build failed because of changes made in `master`

File: UNRELEASED.md [hunk 3]
```diff
@@ -7,6 +7,8 @@
 ### Enhancements
 
 - Updated `Filters` to only show the "More filters" button if necessary ([#2856](https://github.com/Shopify/polaris-react/pull/2856)).
+- Added utilities for parsing video duration (https://polaris.shopify.com/components/images-and-icons/video-thumbnail) ([#2725](https://github.com/Shopify/polaris-react/pull/2725))
```
Comment (kyledurand) [inline UNRELEASED.md:10]: This line can be removed. It already exists in the changelog which is why your build is failing. You might need to remove this line and rebase again 😅

Comment (thayannevls) [reply UNRELEASED.md]: I'm sorry, something went wrong with my rebase. Just fixed it :smile:

Comment (sylvhama) [reply UNRELEASED.md]: There is another conflict in `UNRELEASED.md` 😅

Comment (thayannevls) [reply UNRELEASED.md]: Fixed again :sweat_smile:

Comment (sylvhama) [reply UNRELEASED.md]: I think we are very unlucky today there is again a conflict 😢

Comment (kyledurand) [reply UNRELEASED.md]: Let me know if you want me to get this over the finish line @thayannevls

Happy to help, and sorry about all of the conflicts

Comment (thayannevls) [reply UNRELEASED.md]: @shioyama Yes :cry: I fixed again lol

@kyledurand Do you think we can already merge this PR?

Comment (sylvhama) [reply UNRELEASED.md]: Now the CI is failing because of `duplicate-changelog.test.js`, let me know if I can help to fix that.

**Comment by sylvhama** [comment]
@thayannevls it seems the CI has some problems, I will follow up with Polaris people to find a solution!

**Pull request merged**